assignments/models.py

```diff
@@ -121,3 +121,55 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class SchoolClass(models.Model):
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        verbose_name = _('class')
+        verbose_name_plural = _('classes')
+
+    def __str__(self):
+        return self.name
+
+
+class School(models.Model):
+    name = models.CharField(max_length=255)
+    assignment = models.ForeignKey(Assignment, related_name='schools')
+    classes = models.ManyToManyField(SchoolClass, related_name='schools')
+
+    class Meta:
+        verbose_name = _('school')
```

Comment:
I haven't set verbose names for models that are just used for storing data, like answers (no forms in admin)